docs/preview/02-Features/02-message-handling/01-service-bus.md

```diff
@@ -491,10 +487,6 @@ public class Startup
                 // this job instance in a multi-instance deployment (default: guid).
                 options.JobId = Guid.NewGuid().ToString();
 
-                // The format of the message correlation used when receiving Azure Service Bus messages.
-                // (default: W3C).
-                options.Routing.Correlation.Format = MessageCorrelationFormat.Hierarchical;
```

Comment:
Removes also the mentioning of this format in the feature docs, until we have a clear understanding on how to integrate with OpenTelemetry.
W3C is still default, now.